ror-tools-core/build.gradle

```diff
@@ -114,25 +96,21 @@ compileScala {
     options.compilerArgs << "-Ywarn-macros:after"
 }
 
-tasks.withType(ScalaCompile) {
```

Comment:
maybe this causes the patching step doesn't work in integration tests?

Reply:
Should be fixed now